cnames_active.js

```diff
@@ -3574,7 +3575,7 @@ var cnames_active = {
   "zyh": "zyh-chopper.github.io/zyh",
   "zykj": "cname.vercel-dns.com", // noCF
   "zyx": "zyx.alwaysdata.net",
-  "zyy": "zyyou.github.io/notes"
+  "zyy": "zyyou.github.io/notes",
```

Comment:
```suggestion
  "zyy": "zyyou.github.io/notes"
```
Please revert this unrelated change.

Reply:
i don't know how to revert this change thought it seems no change at all.